Generator/Sources/Generator.swift

```diff
@@ -0,0 +1,189 @@
+                throw GeneratorError.attributeNameNotFound(attribute.id)
+            }
+            namespace.attributes[attributeName] = attribute
+        }
+
+        // Generate files
+        var topLevelNamespaces = namespaceTree.subNamespaces.values.sorted(by: { $0.id < $1.id })
+
+        if let namespaces = namespaces {
+            let namespaceSet = Set(namespaces.split(separator: ",").map { String($0) })
+            // Filter to only include the specified namespace
+            topLevelNamespaces = topLevelNamespaces.filter { namespaceSet.contains($0.id) }
+        }
+
+        // Generate individual target files
+        let renderers: [FileRenderer] = [
+            OTelAttributeRenderer(),
+            SpanAttributeRenderer(),
+        ]
+        for renderer in renderers {
+            let directory = "\(repoDirectory)Sources/\(renderer.targetDirectory)Generated/"
+            if fileManager.fileExists(atPath: directory) {
+                try fileManager.removeItem(atPath: directory)
+            }
+            try fileManager.createDirectory(atPath: directory, withIntermediateDirectories: true)
+
+            for namespace in topLevelNamespaces {
+                let filePath = directory + "\(renderer.fileNamePrefix)\(namespace.memberName).swift"
+                var fileContents = generatedFileHeader
+                fileContents += try renderer.renderFile(namespace)
+
+                try? fileManager.removeItem(atPath: filePath)
+                guard let fileData = fileContents.data(using: .utf8) else {
+                    print("File data for \(filePath) could not be encoded as UTF8")
+                    continue
+                }
+                fileManager.createFile(atPath: filePath, contents: fileData)
+                print("Wrote file \(filePath).")
+            }
+        }
+    }
+}
+
+/// A recursive type that is used to collect attributes. Namespaces may contain sub-namespaces.
+/// The attribute and subNamespace properties act as a prefix tree on the attribute and namespace ids.
+class Namespace {
+    // The full dot-separated path of the namespace. For example `http.request`
+    let id: String
+    // The final entry in the dot-separated path
+    let name: String
+    var attributes: [String: Attribute]
+    var subNamespaces: [String: Namespace]
+
+    init(id: String, attributes: [String: Attribute] = [:], subNamespaces: [String: Namespace] = [:]) {
+        self.id = id
+        self.attributes = attributes
+        self.subNamespaces = subNamespaces
+        self.name = String(id.split(separator: ".").last ?? "")
+    }
+
+    var typeName: String {
+        nameGenerator.swiftTypeName(for: name)
+    }
+
+    var memberName: String {
+        nameGenerator.swiftMemberName(for: name)
+    }
+}
+
+let generatedFileHeader = """
+// DO NOT EDIT. This file is generated automatically. See README for details.
+
+// swiftlint:disable all
```

Comment:
```suggestion
// swift-format-ignore-file
```
I'm planning on adding swift-format to the project (#5) instead of SwiftLint. Do we want to use its ignore pattern here already?

Reply:
Oh perfect! The swiftlint exclusion was mainly to address:

1. Lowercase type names
1. Type nesting deeper than 1 level
1. Long documentation lines

I actually think that running the generated files through a `swift-format` pass after generation could address a lot of the whitespace inconsistencies that are pretty tricky to get right in the generation logic.

Reply:
> running the generated files through a swift-format pass after generation

Sounds good to me. I merged the swift-format setup into main just now, so feel free to rebase and already run swift-format on your PR.

Reply:
Awesome, thanks! I rebased on that branch and brought everything in line with the formatting.

Reply:
Nice! Should we remove the `swiftlint-ignore` comment then?

Reply:
Oh yeah, sorry about that, I glanced the format lint in `check-swift-format.sh`, and absentmindedly thought we might have left in swiftlint. Thanks for double checking me - should be fixed in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/aefda5d01acab9f41bc9b137afd64c321c9b9101